packages/story-editor/src/components/panels/design/link/link.js

```diff
@@ -63,6 +69,30 @@ const Error = styled.span`
   color: ${({ theme }) => theme.colors.fg.negative};
 `;
 
+const CheckboxWrapper = styled.div`
+  display: flex;
+  padding: 9px 0;
+`;
+
+const Label = styled.label`
+  margin-left: 12px;
+`;
+
+const CheckboxContainer = styled.div`
+  margin: 12px 0;
+`;
+
+const StyledText = styled(Text)`
+  color: ${({ theme }) => theme.colors.fg.secondary};
+  padding: 8px 0;
+`;
+
+const LinkTypes = [
+  { key: 'nofollow', title: __('No follow', 'web-stories') },
+  { key: 'ugc', title: __('User-generated content', 'web-stories') },
```

Comment:
As per discussion on #4440 we don't want a checkbox for ugc

Reply:
Removed.